### PR TITLE
Add New Featured Artists: Nitro Fun & Rezonate newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
3️⃣ 0️⃣ 0️⃣ 🎉

### CODE REVIEW

**Comment by Ephemeralis** [comment]
Currently waiting on a banner image.

File: news/2022-09-07-new-featured-artists-nitro-fun-rezonate.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+date: 2022-09-07 10:00:00 +0000
+---
+
+A duo of [**Monstercat**](https://osu.ppy.sh/beatmaps/artists/255) talents have made their homes in our Featured Artists library!
+
+![](/wiki/shared/news/2022-09-07-new-featured-artists-nitro-fun-rezonate/nitro-fun-rezonate-banner.jpg)
+
+Ultra-talented producers **Nitro Fun** and **Rezonate** offer two distinct takes on EDM.
+
+**Nitro Fun** emphasizes literal *fun* through high-tempo and high-energy with a style that especially appeals to video game audiences, as seen through feature spots in *Sonic the Hedgehog*, *Forza Horizon*, and *Just Shapes and Beats.* **Rezonate**, however, leans towards the side of pure *relaxation*, producing longer songs at lower tempos that evoke a sense of grandiose exploration.
```
Comment (Walavouchey) [inline news/2022-09-07-new-featured-artists-nitro-fun-rezonate.md:13]:
```suggestion
**Nitro Fun** emphasizes literal *fun* through high tempo and high energy with a style that especially appeals to video game audiences, as seen through feature spots in *Sonic the Hedgehog*, *Forza Horizon*, and *Just Shapes and Beats*. **Rezonate**, however, leans towards the side of pure *relaxation*, producing longer songs at lower tempos that evoke a sense of grandiose exploration.
```

File: news/2022-09-07-new-featured-artists-nitro-fun-rezonate.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+A combined **6** tracks from both of these artists are ready for mapping! Take your pick from [**Nitro Fun**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/299) or [**Rezonate**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/300).
+
+Need some mapping inspiration? Take a peek at this duo-feature video showcasing [Nitro Fun - Cheat Codes VIP](https://osu.ppy.sh/beatmapsets/711420) and [Rezonate - The Phoenix](https://osu.ppy.sh/beatmapsets/711926), or play through some maps for yourself.
```
Comment (Walavouchey) [inline news/2022-09-07-new-featured-artists-nitro-fun-rezonate.md:17]:
```suggestion
Need some mapping inspiration? Take a peek at this duo feature video showcasing [Nitro Fun - Cheat Codes VIP](https://osu.ppy.sh/beatmapsets/711420) and [Rezonate - The Phoenix](https://osu.ppy.sh/beatmapsets/711926), or play through some maps for yourself.
```

File: news/2022-09-07-new-featured-artists-nitro-fun-rezonate.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Take one last adventure through [this osu! map](https://osu.ppy.sh/beatmapsets/671617) hosted by [Fursum](https://osu.ppy.sh/users/4865030).
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/300/Prelude%20EP%2FRezonate%20-%20The%20Journey.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+As of today, we've officially hit **300 total Featured Artists!** Celebrate this awesome milestone by one of the songs from [**Nitro Fun**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/299) or [**Rezonate**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/300) (which come in pre-timed beatmap templates, by the way).
```
Comment (Walavouchey) [inline news/2022-09-07-new-featured-artists-nitro-fun-rezonate.md:61]:
```suggestion
As of today, we've officially hit **300 total Featured Artists!** Celebrate this awesome milestone with one of the songs from [**Nitro Fun**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/299) or [**Rezonate**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/300) (which come in pre-timed beatmap templates, by the way).
```

File: news/2022-09-07-new-featured-artists-nitro-fun-rezonate.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+That's all for today's double-release! Keep an eye on the [@osugame Twitter](https://twitter.com/osugame) for anything and everything circle-related, along with our next Featured Artist.
```
Comment (Walavouchey) [inline news/2022-09-07-new-featured-artists-nitro-fun-rezonate.md:63]:
```suggestion
That's all for today's double release! Keep an eye on the [@osugame Twitter](https://twitter.com/osugame) for anything and everything circle-related, along with our next Featured Artist.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
:)

**Comment by Ephemeralis** [comment]
I've added an attempt at a header image for this, it can go out if judged appropriate.

**Pull request merged**